op-challenger/game/fault/contracts/oracle.go

```diff
@@ -150,12 +150,15 @@ func (c *PreimageOracleContract) GetActivePreimages(ctx context.Context, blockHa
 		idents = append(idents, c.decodePreimageIdent(result))
 	}
 
-	// Fetch the metadata for each preimage
+	return c.GetProposalMetadata(ctx, block, idents)
+}
+
+func (c *PreimageOracleContract) GetProposalMetadata(ctx context.Context, block batching.Block, idents []gameTypes.LargePreimageIdent) ([]gameTypes.LargePreimageMetaData, error) {
```

Comment:
Would it be worth using varargs here?
```suggestion
func (c *PreimageOracleContract) GetProposalMetadata(ctx context.Context, block batching.Block, idents ...gameTypes.LargePreimageIdent) ([]gameTypes.LargePreimageMetaData, error) {
```